### PR TITLE
Add GoReleaser plugin

### PR DESCRIPTION
**Pull request by kforsthoevel**
This change will add the [plugin](https://github.com/kforsthoevel/asdf-goreleaser) for [GoReleaser](https://goreleaser.com/).

# Checklist

- [ ] Provide a short description and link to your plugin.
- [ ] Make sure you CI tests for your plugin and they are green.
- [ ] `asdf-plugins` CI sanity checks are green on your PullRequest.

### CODE REVIEW

**Comment by kforsthoevel** [comment]
I don't understand why the check has failed. I have added a new line for my plugin to the README.md. Could anyone advise please?

File: plugins/goreleaser [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/kforsthoevel/asdf-plugins.git
```
Comment (smorimoto) [inline plugins/goreleaser:1]:
```suggestion
repository = https://github.com/kforsthoevel/asdf-goreleaser.git
```

**Comment by smorimoto** [comment]
​I think the badge should be this one.
```md
![Main workflow](https://github.com/kforsthoevel/asdf-goreleaser/workflows/Main%20workflow/badge.svg?branch=master)
```

**Comment by kforsthoevel** [comment]
Thanks you so much for your help @smorimoto ❤️

**Review by smorimoto — APPROVED** [review]

**Comment by smorimoto** [comment]
Anytime :) Thanks for your PR @kforsthoevel!

**Pull request merged**